refactor: Extract UI management into main/core lib

maciejBart99 opened the pull request.

maciejBart99 committed:
refactor: Extract UI management into main/core lib

MaciejSikorski commented on libs/main/core/src/lib/main-core.module.ts:
```diff
@@ -0,0 +1,29 @@
+    MatMenuModule,
+    MatIconModule,
+    SharedUiUserBarModule,
+    RouterModule.forChild([])],
```
just RouterModule

MaciejSikorski commented on libs/main/core/src/lib/todo-app/todo-app.component.ts:
```diff
@@ -0,0 +1,24 @@
+  selector: 'todo-application-todo-app',
+  templateUrl: './todo-app.component.html'
+})
+export class TodoAppComponent {
```
MainComponent

MaciejSikorski commented on apps/todo/src/app/app.module.ts:
```diff
@@ -3,18 +3,25 @@ import { NgModule } from '@angular/core';
-    maxAge: 25
-  })],
+  imports: [BrowserModule,
+            BrowserAnimationsModule,
```
Move that "forRoot" modules to the core library dedicated for the todo-app

maciejBart99 committed:
refactor: Name changes and new core module

-Rename main/core to main/shell
-Put all libs in todo-app scope
-Add core moduel for store and router configuration
